data/yaml/page-furniture/sidebar.yaml

```diff
@@ -100,175 +157,17 @@ items:
         link: /docs/asset-tracking/example-apps
       - label: "Using the SDKs"
         link: /docs/asset-tracking/using-the-sdks
-  - label: "Ably accounts and apps"
-    link: ""
-    items:
-      - label: "Programmatic management with Control API"
```

Comment:
Do we have content for Terraform? Might that go here?

Reply:
We don't have anything in the docs yet, no. But agreed that this would likely be the best place for it when we do.